realm/realm-library/src/main/java/io/realm/internal/sync/permissions/ObjectPermissionsModule.java

```diff
@@ -0,0 +1,13 @@
+package io.realm.internal.sync.permissions;
+
+import io.realm.annotations.RealmModule;
+import io.realm.sync.Subscription;
+
+/**
+ * Realm model classses that are always part of Query-based Realms
+ */
+@RealmModule(library = true, classes = {
+        Subscription.class
+})
+public class ObjectPermissionsModule {
```

Comment:
Are we still keeping this permission model for QBS though? I thought all permission including object-level permission are going to be on the Stitch side as well?

Reply:
No, I'm going to remove this when I remove subscriptions. I just went with minimal changes when removing the permissions.